src/monitoring_service/states.py

```diff
@@ -178,6 +180,7 @@ def from_balance_proof(
         )
 
     def sign(self, priv_key: str) -> 'MonitorRequest':
+        signer = LocalSigner(private_key=to_bytes(hexstr=priv_key))
```

Comment:
Now `signer` is a `LocalSigner` instance in some places and an Ethereum address of the signing party in other places. This can cause confusion or name clashes. But I also don't have a good suggestion, right now.